.github/workflows/contributing.yml

```diff
@@ -0,0 +1,44 @@
+name: contributing
```

Comment:
should we clean this up remove all the commented out code?

Reply:
of course, sorry about that. thanks for noticing! Now running the action on multiple python versions in parallel.